### PR TITLE
Convert docs to MD

### PR DESCRIPTION
**Pull request by trestletech**
Automatically converted the docs using https://roxygen2md.r-lib.org/

Connected to #2475 

I followed the process recommended for large packages in https://roxygen2md.r-lib.org/ and you can see the commits as I went

It went OK, but there were a few places that generated some concern:

1. There are three instances in the MD docs where % signs aren't getting escaped as they should in the rendered Rd. This is a silent failure and is interpreted as a comment in the Rd, so the rest of the line is just silently omitted. The current workaround is just to manually escape them, but that doesn't install confidence. See 09dee96 and 633817e.
2. the `parseQueryString` function has too many markdown-looking characters and the parser kept getting confused. I fixed it by opting out of MD for that one function.
3. The last step modifies the links and the docs state:

> This is the trickiest part of the entire conversion. Due to the staged approach, the changeset is reduced to a minimum. Review carefully, luckily R CMD check catches bad links introduced in this stage.

There are thousands of changes here, so I did not review manually.

### Alternative Approaches

 - We could stay with RD syntax in Roxygen, no-op.
 - We could opt-in to RD one function at a time at a more manually reviewable pace using the `@md` function. Effectively, we'd just take the first commit here and then roll the functions over to MD over time.

### Testing Strategy

... 🤷‍♂️. I don't know that there's a meaningful way to review this one other than following along the commits. I originally hoped that the Rd files wouldn't change, but that turned out not to be true. There are lots of whitespace and line break changes plus a big swath of link changes as we change the way we link function references.

I thought that pkgdown might be another format that would have fewer diffs, but it also has lots of whitespace diffs and all the function link changes overwhelm the diff.

Barrett had the good idea of looking at the GitHub diff and clicking "settings" > Ignore Whitespace and Unified. When you do that, you can scroll all the way down and start clicking to expand the Rd files individually. Your eyes can start to get trained on the patterns that are normal (expanded green text for longer function hyperlinks). He did one pass initially and found a few bugs that way. We paired through them together, then we both did separate passes individually and feel good about it.

Session info:
<details>
> devtools::session_info()
─ Session info ───────────────────────────────────────────────────────────────────────────────
 setting  value                       
 version  R version 3.5.2 (2018-12-20)
 os       macOS Mojave 10.14.5        
 system   x86_64, darwin15.6.0        
 ui       RStudio                     
 language (EN)                        
 collate  en_US.UTF-8                 
 ctype    en_US.UTF-8                 
 tz       America/Chicago             
 date     2019-06-19                  

─ Packages ───────────────────────────────────────────────────────────────────────────────────
 ! package     * version    date       lib source                               
   assertthat    0.2.1      2019-03-21 [1] CRAN (R 3.5.2)                       
   backports     1.1.4      2019-04-10 [1] CRAN (R 3.5.2)                       
   brew          1.0-6      2011-04-13 [1] CRAN (R 3.5.0)                       
   callr         3.2.0      2019-03-15 [1] CRAN (R 3.5.2)                       
   cli           1.1.0      2019-03-19 [1] CRAN (R 3.5.2)                       
   clisymbols    1.2.0      2017-05-21 [1] CRAN (R 3.5.0)                       
   commonmark    1.7        2018-12-01 [1] CRAN (R 3.5.0)                       
   crayon        1.3.4      2017-09-16 [1] CRAN (R 3.5.0)                       
   curl          3.3        2019-01-10 [1] CRAN (R 3.5.2)                       
   desc          1.2.0      2018-05-01 [1] CRAN (R 3.5.0)                       
   devtools      2.0.2      2019-04-08 [1] CRAN (R 3.5.2)                       
   digest        0.6.19     2019-05-20 [1] CRAN (R 3.5.2)                       
   enc           0.2.0      2018-03-03 [1] CRAN (R 3.5.0)                       
   evaluate      0.14       2019-05-28 [1] CRAN (R 3.5.2)                       
   fastmap       0.0.0.9001 2019-06-14 [1] Github (r-lib/fastmap@91321f1)       
   fs            1.3.1      2019-05-06 [1] CRAN (R 3.5.2)                       
   glue          1.3.1      2019-03-12 [1] CRAN (R 3.5.2)                       
   htmltools     0.3.6.9003 2019-06-14 [1] Github (rstudio/htmltools@eea9211)   
   httpuv        1.5.1.9000 2019-06-14 [1] Github (rstudio/httpuv@da0a7ef)      
   jsonlite      1.6        2018-12-07 [1] CRAN (R 3.5.0)                       
   knitr         1.23       2019-05-18 [1] CRAN (R 3.5.2)                       
   later         0.8.0.9000 2019-06-14 [1] Github (r-lib/later@55f63f3)         
   lazyeval      0.2.2      2019-03-15 [1] CRAN (R 3.5.2)                       
   magrittr      1.5        2014-11-22 [1] CRAN (R 3.5.0)                       
   MASS          7.3-51.1   2018-11-01 [1] CRAN (R 3.5.2)                       
   memoise       1.1.0      2017-04-21 [1] CRAN (R 3.5.0)                       
   mime          0.7        2019-06-11 [1] CRAN (R 3.5.2)                       
   packrat       0.5.0      2018-11-14 [1] CRAN (R 3.5.2)                       
   pillar        1.4.1      2019-05-28 [1] CRAN (R 3.5.2)                       
   pkgbuild      1.0.3      2019-03-20 [1] CRAN (R 3.5.2)                       
   pkgconfig     2.0.2      2018-08-16 [1] CRAN (R 3.5.0)                       
   pkgdown       1.3.0.9100 2019-06-19 [1] Github (r-lib/pkgdown@b17fc6b)       
   pkgload       1.0.2      2018-10-29 [1] CRAN (R 3.5.0)                       
   prettyunits   1.0.2      2015-07-13 [1] CRAN (R 3.5.0)                       
   processx      3.3.1      2019-05-08 [1] CRAN (R 3.5.2)                       
   promises      1.0.1      2018-04-13 [1] CRAN (R 3.5.0)                       
   ps            1.3.0      2018-12-21 [1] CRAN (R 3.5.0)                       
   purrr         0.3.2      2019-03-15 [1] CRAN (R 3.5.2)                       
   R6            2.4.0      2019-02-14 [1] CRAN (R 3.5.2)                       
   Rcpp          1.0.1      2019-03-17 [1] CRAN (R 3.5.2)                       
   remotes       2.0.4      2019-04-10 [1] CRAN (R 3.5.2)                       
   rex           1.1.2      2017-10-19 [1] CRAN (R 3.5.0)                       
   rlang         0.3.4      2019-04-07 [1] CRAN (R 3.5.2)                       
   rmarkdown     1.13       2019-05-22 [1] CRAN (R 3.5.2)                       
   roxygen2      6.1.1      2018-11-07 [1] CRAN (R 3.5.0)                       
   roxygen2md    1.0.0      2019-06-19 [1] Github (r-lib/roxygen2md@05d3729)    
   rprojroot     1.3-2      2018-01-03 [1] CRAN (R 3.5.0)                       
   rstudioapi    0.10       2019-03-19 [1] CRAN (R 3.5.2)                       
   sessioninfo   1.1.1      2018-11-05 [1] CRAN (R 3.5.0)                       
 P shiny       * 1.3.2.9000 2019-06-19 [?] local                                
   sourcetools   0.1.7      2018-04-25 [1] CRAN (R 3.5.0)                       
   stringi       1.4.3      2019-03-12 [1] CRAN (R 3.5.2)                       
   stringr       1.4.0      2019-02-10 [1] CRAN (R 3.5.2)                       
   testthat      2.1.1      2019-04-23 [1] CRAN (R 3.5.2)                       
   tibble        2.1.3      2019-06-06 [1] CRAN (R 3.5.2)                       
   usethis       1.5.0.9000 2019-06-19 [1] Github (r-lib/usethis@2baa06f)       
   withr         2.1.2      2018-03-15 [1] CRAN (R 3.5.0)                       
   xfun          0.7        2019-05-14 [1] CRAN (R 3.5.2)                       
 V xml2          1.2.0      2019-06-19 [1] Github (HenrikBengtsson/xml2@618bc74)
   xtable        1.8-4      2019-04-21 [1] CRAN (R 3.5.2)                       

[1] /Library/Frameworks/R.framework/Versions/3.5/Resources/library

 V ── Loaded and on-disk version mismatch.
 P ── Loaded and on-disk path mismatch.
</details>

### CODE REVIEW

File: man/Progress.Rd [hunk 1]
```diff
@@ -57,28 +57,28 @@ If you want to use the old styling (for example, you may have used customized
 CSS), you can use \code{style="old"} each time you call
 \code{Progress$new()}. If you don't want to set the style each time
 \code{Progress$new} is called, you can instead call
-\code{\link{shinyOptions}(progress.style="old")} just once, inside the server
+\code{[shinyOptions](progress.style="old")} just once, inside the server
```
Comment (schloerke) [inline man/Progress.Rd]: [ ] link failed

File: man/fileInput.Rd [hunk 2]
```diff
@@ -14,14 +14,14 @@ fileInput(inputId, label, multiple = FALSE, accept = NULL,
 \item{label}{Display label for the control, or \code{NULL} for no label.}
 
 \item{multiple}{Whether the user should be allowed to select and upload
-multiple files at once. \bold{Does not work on older browsers, including
+multiple files at once. \strong{Does not work on older browsers, including
```
Comment (schloerke) [inline man/fileInput.Rd]: is this the same?

File: man/plotOutput.Rd [hunk 3]
```diff
@@ -81,35 +81,35 @@ for the output}
...
-Render a \code{\link{renderPlot}} or \code{\link{renderImage}} within an
+Render a \code{\link[=renderPlot]{renderPlot()}} or \code{\link[=renderImage]{renderImage()}} within an
 application page.
 }
 \note{
 The arguments \code{clickId} and \code{hoverId} only work for R base
-  graphics (see the \pkg{\link[graphics:graphics-package]{graphics}} package). They do not work for
-  \pkg{\link[grid:grid-package]{grid}}-based graphics, such as \pkg{ggplot2},
-  \pkg{lattice}, and so on.
+graphics (see the \pkg{[graphics][graphics::graphics-package]} package). They do not work for
```
Comment (schloerke) [inline man/plotOutput.Rd]: these two lines seem off

File: man/reactlog.Rd [hunk 4]
```diff
@@ -55,11 +55,11 @@ of the source code of your reactive expressions and observers.
 \section{Functions}{
 \itemize{
 \item \code{reactlog}: Return a list of reactive information.  Can be used in conjunction with
-\code{reactlog::\link[reactlog]{reactlog_show}} to later display the reactlog graph.
+\code{reactlog::[reactlog_show][reactlog::reactlog_show]} to later display the reactlog graph.
```
Comment (schloerke) [inline man/reactlog.Rd]: bug

File: man/session.Rd [hunk 5]
```diff
@@ -5,174 +5,174 @@
...
 }
 \item{isClosed()}{A function that returns \code{TRUE} if the client has
-  disconnected.
+disconnected.
 }
 \item{ns(id)}{
-  Server-side version of \code{ns <- \link{NS}(id)}. If bare IDs need to be
-  explicitly namespaced for the current module, \code{session$ns("name")}
-  will return the fully-qualified ID.
+Server-side version of \code{ns <- [NS](id)}. If bare IDs need to be
```
Comment (schloerke) [inline man/session.Rd]: bug

File: man/withProgress.Rd [hunk 6]
```diff
@@ -84,7 +84,7 @@ If you want to use the old styling (for example, you may have used customized
 CSS), you can use \code{style="old"} each time you call
 \code{withProgress()}. If you don't want to set the style each time
 \code{withProgress} is called, you can instead call
-\code{\link{shinyOptions}(progress.style="old")} just once, inside the server
+\code{[shinyOptions](progress.style="old")} just once, inside the server
```
Comment (schloerke) [inline man/withProgress.Rd]: bug

**Comment by trestletech** [comment]
It looks like the two manually-adjusted backslashes were the only two that didn't occur in \code blocks. All the rest are now in backticks. So the MD renderer doesn't escape % signs unless they're in code.

File: man/fileInput.Rd (hunk 2 above)
Comment (trestletech) [reply man/fileInput.Rd]: Yes, I believe so.

**Comment by trestletech** [comment]
@schloerke I just finished my pass through the diff and all LGTM.

**Pull request merged**